docs/readme_chunks/generate_frontpage_documents.py

```diff
@@ -0,0 +1,112 @@
+import os
+import sys
+
+class DocGenerator:
+    """A Class to represent a documentation generator
+
+    Attributes: None
+
+    Methods
+    _________
+    get_chunk_paths()
+        returns the chunks that have been made
+    """
+
+
+    #CHUNKED DOCUMENTS
+    INTRO_DOC = "./docs/readme_chunks/introduction.md"
```

Comment:
Dealing with paths can be hard - I would suggest using Pathlib to deal with all the paths-related operations and have absolute paths